doc/source/serve/monitoring.md

```diff
@@ -82,18 +111,21 @@ class Silenced:
         logger.setLevel(logging.ERROR)
 ```
 
-This will prevent the replica INFO-level logs from being written to STDOUT or to files on disk.
+This controls which logs are written to STDOUT or files on disk.
 You can also use your own custom logger, in which case you'll need to configure the behavior to write to STDOUT/STDERR, files on disk, or both.
 
-### Tutorial: Ray Serve with Loki
+For a detailed overview of logging in Ray, see [Ray Logging](ray-logging).
 
-Here is a quick walkthrough of how to explore and filter your logs using [Loki](https://grafana.com/oss/loki/).
-Setup and configuration is very easy on Kubernetes, but in this tutorial we'll just set things up manually.
+(serve-logging-loki)=
+### Filtering logs with Loki
 
-First, install Loki and Promtail using the instructions on <https://grafana.com>.
-It will be convenient to save the Loki and Promtail executables in the same directory, and to navigate to this directory in your terminal before beginning this walkthrough.
+You can explore and filter your logs using [Loki](https://grafana.com/oss/loki/).
+Setup and configuration is straightforward on Kubernetes, but as a tutorial, let's set up Loki manually.
 
```

Comment:
Is "setup and configuration" considered a singular compound subject (like ham and cheese is great for breakfast) in this context? If not perhaps change to

"Setup and configuration are straightforward on Kubernetes."